code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -151,19 +151,9 @@
 	name = "reactive tesla armor"
 	desc = "An experimental suit of armor with sensitive detectors hooked up to a huge capacitor grid, with emitters strutting out of it. Zap."
 	siemens_coefficient = -1
-	var/tesla_power = 25000
-	var/tesla_range = 20
-	var/tesla_flags = TESLA_MOB_DAMAGE | TESLA_OBJ_DAMAGE
-
-/obj/item/clothing/suit/armor/reactive/tesla/dropped(mob/user)
-	..()
-	if(istype(user))
-		user.flags_1 &= ~TESLA_IGNORE_1
-
-/obj/item/clothing/suit/armor/reactive/tesla/equipped(mob/user, slot)
-	..()
-	if(slot_flags & slot) //Was equipped to a valid slot for this item?
-		user.flags_1 |= TESLA_IGNORE_1
```

Comment:
This isn't really fixing the issue, it's just sweeping it under the rug until something else gets added that causes it.

Reply:
That's true. It is kinda a bandage fix. Would you like me to make it a trait, or is that too much of a operation to add to Tesla code? I could make it a bitflag tied to a trait if checking has_trait is too expensive.

Reply:
I've re-added the relevant code, I await the impending merge conflict.